build/npm/gyp/yarn.lock

```diff
@@ -301,7 +301,7 @@ glob@^7.1.3, glob@^7.1.4:
     fs.realpath "^1.0.0"
     inflight "^1.0.4"
     inherits "2"
-    minimatch "^3.0.4"
+    minimatch "^3.0.5"
```

Comment:
This shouldn't change. If you re-run yarn does it keep this? All this is saying is what the dependency requested by the glob package is, so it's fine to leave as `^3.0.4` since the actual resolution is done below:

```
minimatch@^3.0.4:
  version "3.1.2"
```

and you can see it's already resolving to 3.1.2 so shouldn't need to be updated at all.